integration/nwo/fabric/topology/ds.go

```diff
@@ -22,18 +22,23 @@ type Logging struct {
 	Format string `yaml:"format,omitempty"`
 }
 
+type UserSpec struct {
+	Name string `yaml:"Name"`
+	HSM  bool   `yaml:"HSM"`
```

Comment:
```suggestion
	UseHSM  bool   `yaml:"HSM"`
```

Reply:
but what about the yaml part? Why changing?